analysis_driver/config.py

```diff
@@ -143,7 +143,9 @@ def _etc_config(config_file):
 default = EnvConfiguration(
     [
         os.getenv('ANALYSISDRIVERCONFIG'),
-        os.path.expanduser('~/.analysisdriver.yaml')
+        os.path.expanduser('~/.analysisdriver.yaml'),
+        os.path.join(os.path.dirname(os.path.dirname(__file__)), 'etc', 'example_analysisdriver.yaml')
+
```

Comment:
Is this for tests? I think we had `etc` in the search path before, but removed it because the example config would never be suitable when actually running the pipeline